tests/ws/driver_test.go

```diff
@@ -6,9 +6,9 @@ import (
 	"database/sql"
 	"os"
 	"testing"
-	"time"
 
 	_ "github.com/libsql/libsql-client-go/libsql"
+	_ "modernc.org/sqlite"
```

Comment:
Why is this needed in this commit?

Reply:
it is not needed, I have removed it. 

It slipped in by mistake. I am running the tests against both libsql and sqlite3. Also, we add github workflow, then its a good idea to test the code against both in the CI

Reply:
Thanks!